app/presenters/summary/sections/case_details.rb

```diff
@@ -36,13 +36,13 @@ def answers
 
           Components::ValueAnswer.new(
             :appeal_financial_circumstances_changed, kase.appeal_financial_circumstances_changed,
-            show: original_app_submitted?,
+            show: original_app_submitted? && !crime_application.cifc?,
```

Comment:
Would it be cleaner to put a guard in the `original_app_submitted` and `financial_circumstances_changed` questions?